src/main/flight/feedforward.c

```diff
@@ -59,7 +59,7 @@ void feedforwardInit(const pidProfile_t *pidProfile)
     }
 }
 
-FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforwardAveraging_t feedforwardAveraging)
+FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforwardAveraging_t feedforwardAveraging, const float setpoint)
```

Comment:
If you want this to work in angle mode, we will require a specific, dedicated feedforward calculation while in angle mode.  That's because rcCommand is not relevant, nor do we require jitter reduction again, having done it in pid.c when we created the setpoint.  Additionally, the setpoint we are sending here is already smoothed, and it is smooth enough that no further smoothing is needed.

Reply:
I'll copy your example from anglePR mod V2